src/init.cpp

```diff
@@ -1563,7 +1563,7 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
                 chainman.m_total_coinstip_cache = nCoinCacheUsage;
                 chainman.m_total_coinsdb_cache = nCoinDBCache;
 
-                UnloadBlockIndex(node.mempool.get(), chainman);
+                UnloadBlockIndex(chainman);
```

Comment:
Are we sure that there is no need to reset the mempool here? Could anything have been added to it further in this loop?

Reply:
Yes, otherwise I wouldn't have opened the pull request. You probably wanted to say that the review effort is not worth it?

Reply:
No, I just wanted to make sure you hadn't missed the fact that there is a loop here.

Reply:
Ah ok. I might reopen some other day, unless someone beats me to it.